bento/src/main/java/com/yelp/android/bento/components/ListComponent.java

```diff
@@ -253,6 +255,38 @@ public int getSpanSize(int position) {
         };
     }
 
+    @Override
+    public void onItemsMoved(
```

Comment:
Should we make this `final` here?

Reply:
Eh, I can't imagine a reason someone would want to override this.